serialzy/serializers/exception.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import sys
+from inspect import isclass
+from typing import Any, BinaryIO, Callable, Dict, Type, Union, Optional
+
+from packaging import version  # type: ignore
+from serialzy.api import Schema, VersionBoundary
+from serialzy.base import DefaultSchemaSerializerByValue
+from serialzy.utils import cached_installed_packages
+
+_LOG = logging.getLogger(__name__)
+
+
+class ExceptionSerializer(DefaultSchemaSerializerByValue):
+    DATA_FORMAT = "raw_exception"
+
+    def _serialize(self, obj: BaseException, dest: BinaryIO) -> None:
+        from tblib import pickling_support  # type: ignore
+        pickling_support.install()
+        import pickle  # type: ignore
+        pickle.dump(sys.exc_info(), dest)
```

Comment:
I'm not sure it is a rigth place to call `sys.exc_info()`.
What if there will be a new exception between this call and the original exception?
I believe you need to work with exc_info from the start and call it near the original exception.